packages/adapter/package.json

```diff
@@ -7,6 +7,8 @@
   "author": "NodeFactory <info@nodefactory.io>",
   "license": "Apache-2.0",
   "private": false,
+  "homepage": "https://github.com/NodeFactoryIo/metamask-snap-polkadot/tree/master/packages/adapter",
```

Comment:
@mpetrunic There are two options here:
1) Add direct URL to adapter (must be on the specific branch, e.g. _master_)
2) Add generic URL to root repo